pkg/utils/test/oc/literal.go

```diff
@@ -0,0 +1,64 @@
+package oc
+
+import (
+	"fmt"
+	"strings"
+)
+
+// For oc commands not a part of this package, e.g. oc.Logs, oc.Apply, oc.Delete etc,
+// oc.Literal is a workaround to run those commands.
+
+// ILiteral is an interface for collecting the command string
+type ILiteral interface {
```

Comment:
Could we just call it Literal. 
And call `func Literal` `func Command` and it would read 
```
oc.Command("apply -f foobar.yaml).Run() 
```